nss-emerald-wolverines/BangazonAPI
# Db product controller

# Description

I've added the Product Controller and Product Test. CRUD functionality is working and All tests are passing for me.


## Added Product Controller and Product Test
- [X ] New feature (non-breaking change which adds functionality)



# Testing Instructions for Change Made

git fetch --all
git checkout db-product-controller
Open BangazonAPI in Visual Studio
Execute Product controller
Verify in postman that you are able to post, put, edit, and delete (use http://localhost:5000/api/product as url
In Visual Studio click Test Explorer and run ProductTests (for testing the delete function, you'll have to pick an id number that is in your datatbase range (I have 30 hardcoded in the delete part of the test, you'll need something lower, try a single digit number).



# Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] My changes generate no new warnings
- [X ] I have added test instructions that prove my fix is effective or that my feature works
